avalon_client/avalon_client_sdk/direct/jrpc/work_order_receipt_jrpc_client.py

```diff
@@ -16,17 +16,17 @@
 import logging
 from utility.hex_utils import is_valid_hex_str
 
-from service_client.generic import GenericServiceClient
-from connectors.interfaces.work_order_receipt_interface \
-    import WorkOrderReceiptInterface
-from utility.tcf_types import ReceiptCreateStatus
-from connectors.utils import create_jrpc_response, validate_details
+from avalon_client_sdk.service_client.generic import GenericServiceClient
+from avalon_client_sdk.interfaces.work_order_receipt_client \
+    import WorkOrderReceiptClient
+from avalon_client_sdk.utility.tcf_types import ReceiptCreateStatus
+from avalon_client_sdk.utility.utils import create_jrpc_response
```

Comment:
I guess JSONRPCDispatchException has to be called from handler function, anyway i have to remove it from client because client shouldn't through any exception.